tools/save_pretrained_model.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""
+Save pre-trained model.
+"""
+import tensorflow as tf
+import numpy as np
+
+
+def save_pretrained_model(sess, outputs, feeds, out_dir, model_name="pretrained"):
+    """Save pretrained model and config"""
+    try:
+        import os
+        to_onnx_path = "{}/to_onnx".format(out_dir)
+        if not os.path.isdir(to_onnx_path):
+            os.makedirs(to_onnx_path)
+        saved_model = "{}/saved_model".format(to_onnx_path)
+        inputs_path = "{}/inputs.npy".format(to_onnx_path)
+        pretrained_model_yaml_path = "{}/pretrained.yaml".format(to_onnx_path)
+
+        print("===============Save Frozen Graph========================")
+        print("Save model for tf2onnx: {}".format(to_onnx_path))
+        # save inputs
+        inputs = {}
+        for inp, value in feeds.items():
+            if isinstance(inp, str):
+                inputs[inp] = value
+            else:
+                inputs[inp.name] = value
+        np.save(inputs_path, inputs)
+        print("Saved inputs to {}".format(inputs_path))
+
+        # save graph and weights
+        from tensorflow.saved_model import simple_save
```

Comment:
is there any tf version restriction?

Reply:
at least tf1.9 can work, let me double check it.